### ISSUE TITLE
use classes

### ISSUE THREAD

**Issue by rimmesbe** [DianaKado/prj-rev-bwfs-dasmoto#4]
I'd prefer a class used for this styling, this way we don't lock the span down with specific styling incase we want to use it later in a different manner. Also you can use a descriptive class name to make the html more understandable. 

https://github.com/DianaKado/prj-rev-bwfs-dasmoto/blob/master/Dasmoto/resources/css/styles.css#L25-L28

**Comment by DianaKado** [comment]
So, combining this with your suggestion to use strong tags instead of span tags, would the following code be more suitable? (first and last html brackets <> are missing because of styling markdowns in github)

strong class="price">Starting at $3.00 / brush.</strong
strong class="price">Starting at $2.00 / frame.</strong
strong class="price">Starting at $5.00 / tube.</strong

in the html file
and

.price {
font-weight: bold; 
color: blue;
} 

in the css file.
If I upload the updated files, should I upload it to the master or a branch?

Thanks.

**Comment by rimmesbe** [comment]
Yes that is better, except you won't need font-weight: bold; in the css
since strong tag makes it bold.

**Comment by DianaKado** [comment]
Oh, yes, the thought came up early this morning. Font-weight: bold; in css is redundant with strong as it is with the headings. Good to point out. I made the change.
Thanks.
